vms/platformvm/txs/fee/calculator_api.go

```diff
@@ -0,0 +1,26 @@
+// Copyright (C) 2019-2024, Ava Labs, Inc. All rights reserved.
+// See the file LICENSE for licensing terms.
+
+package fee
+
+import (
+	"github.com/ava-labs/avalanchego/vms/components/verify"
+	"github.com/ava-labs/avalanchego/vms/platformvm/txs"
+)
+
+// Calculator is a wrapper to exposed a more user friendly API than txs.Visitor allows
+// backend is not embedded to avoid exposing unnecessary methods
+type Calculator struct {
```

Comment:
I think we can delete this struct if we go with https://github.com/ava-labs/avalanchego/pull/3145/files#r1661788785 as this will no longer serve the original purpose of hiding the `txs.Visitor` interface if we remove it from the `backend` interface.

Reply:
Done